WIP: initial support Nvidia jetson nano model B

Insei opened the pull request:
for example:
make clean
make build
./eden config add default --devmodel jetson-nano-b
./eden config set default --key eve.tag --value=0.0.0-jetson-wip-nkernel-f30f7db5
./eden config set default --key eve.registry --value=goodmobiledevices
./eden setup
./eden start
./eden eve onboard
e.t.c.

giggsoff commented on pkg/utils/downloaders.go:
```diff
@@ -64,7 +66,7 @@ func (desc UEFIDescription) image(latest bool) (string, error) {
 
 //DownloadEveLive pulls EVE live image from docker
-func DownloadEveLive(eve EVEDescription, uefi UEFIDescription, outputFile string) (err error) {
+func DownloadEveLive(model models.DevModel, eve EVEDescription, uefi UEFIDescription, outputFile string) (err error) {
```
Can we add string with model name into EVEDescription type instead of adding new argument?

Insei replied:
Yes, I also like this option more, but there was a meeting, and it seemed to me that it was better not to do it. Perhaps during the discussion I was misunderstood.

giggsoff commented on pkg/models/jetson-nano-b-sdcard.go:
```diff
@@ -0,0 +1,102 @@
+
+//GetPortConfig returns PortConfig overwrite
+func (ctx *DevModelJetson) GetPortConfig(ssid string, psk string) string {
+	return fmt.Sprintf(`{
```
I think, we can return empty string now here. This place is for WiFi.

Insei replied:
Well, thanks, I will, I didn't notice right away.

giggsoff reviewed (commented):
Please take a look.

Insei commented:
I added a GetTarget () method to the DevModel interface to get the target for each devmodel. By default GetTarget() return empty string.
I added a Target property to the EVEDescription structure that will be used for the -t flag when we generate the eve image.

rvs commented:
This looks pretty good, but shouldn't we wait for the EVE Jetson PR to merge first (because there may be other changes required) before merging this?

Insei commented:
Yes, we need to accept PR in EVE first.
